contribs/gnogenesis/internal/txs/txs_add_packages.go

```diff
@@ -27,16 +28,32 @@ var (
 )
 
 type addPkgCfg struct {
-	txsCfg          *txsCfg
-	deployerAddress string
+	txsCfg                *txsCfg
+	keyName               string
+	gnoHome               string
+	insecurePasswordStdin bool
 }
 
 func (c *addPkgCfg) RegisterFlags(fs *flag.FlagSet) {
 	fs.StringVar(
-		&c.deployerAddress,
-		"deployer-address",
-		defaultCreator.String(),
-		"the address that will be used to deploy the package",
+		&c.keyName,
+		"key-name",
+		"",
+		"The package deployer key name or address",
+	)
+
+	fs.StringVar(
+		&c.gnoHome,
+		"gno-home",
+		gnoenv.HomeDir(),
+		"the gno home directory",
+	)
```

Comment:
you can already override this with the `GNOHOME` env variable that is used in `gnoenv.HomeDir()`

Reply:
fixed on af8615275d48487fe0e78eee949519fd9d0eec44

Reply:
Sorry I wasn't clear, using `gnoenv.HomeDir()` in the code is good to get this value. What I meant is that we can remove the cli flag `--gno-home` since you can pass the environment variable when you call the command to achieve the same behavior.

Reply:
Ahh I got it, I tried that before but the test is using t.Parallel and it does not allow me to set the environment variable. So maybe I can leave it as it is ?

Reply:
Got it, yeah we can leave it as-is, maybe add a comment saying that it's mostly used for testing

Reply:
sure, thanks fixed on 8b27abe41c412cc402125e3e4921f523fec98d9d